xinference/model/llm/llm_family.json

```diff
@@ -3930,5 +4026,281 @@
         "<start_of_turn>"
       ]
     }
+  },
+  {
+    "version": 1,
+    "context_length": 4096,
+    "model_name": "Platypus2-70B-instruct",
```

Comment:
Lower case letters are recommended for model names, as all other names in the xinference system are lower case.
Please change all other places.